driver/node.go

```diff
@@ -211,6 +211,12 @@ func (s *nodeService) nodePublishFilesystemVolume(req *csi.NodePublishVolumeRequ
 		return status.Errorf(codes.Internal, "target device is already formatted with different filesystem: volume=%s, current=%s, new:%s", req.GetVolumeId(), fsType, mountOption.FsType)
 	}
 
+	// avoid duplicate XFS UUIDs
+	if mountOption.FsType == "xfs" {
+		mountOptions = append(mountOptions, "nouuid")
+		s.mounter.Exec.Command("xfs_admin", "-U", "generate", device)
```

Comment:
- this `xfs_admin` exists on the platform without installing any extra pkgs right?

Reply:
Yes, It is available.